### ISSUE TITLE
CakePdf not reading CSS / Also, incorrect css file getting precedence

### ISSUE THREAD

**Issue by dlynnsmith** [FriendsOfCake/CakePdf#159]
I'm having a problem exactly like the one described here: http://stackoverflow.com/questions/27197435/cakepdf-not-reading-css from a year ago on StackOverflow.  CakePdf is producing the pdf for me, but with none of the CSS styling from the default.ctp file.  I have run with both a view pdf output to the screen, and a pdf written directly to a file.  Both behave the same. I have the AppHelper for assetUrl set.  

What else might I be missing?

**Comment by dlynnsmith** [comment]
I found a workaround to this, but it led to a separate question.

Workaround: More specific path to the css file. 
From:
    <?= $this->Html->css('cake.css') ?>
To:
    <?= $this->Html->css(WWW_ROOT . 'css/cake.css') ?>

New problem:  If the default.ctp includes both of the css files provided by Bake (base.css and cake.css), the base.css always get precedence.  This is contrary to the standard that the last file listed will get precedence over earlier ones.  The only way I've been able to get cake.css styling is to have it be the only css listed in the .ctp.  Why would that be so?
